dask/base.py

```diff
@@ -586,7 +586,8 @@ def visualize(
     Parameters
     ----------
     args : object
-        Any number of objects. If it is a dask object, its associated graph
+        Any number of objects. If it is a dask collection - That is Dataframe, Array,
```

Comment:
I think @pavithraes was suggesting that this should be "For example, Dataframe, Array,"

Reply:
As you want but I was applying what you said to me earlier: 
> "That is" is a good replacement for i.e. and "For example" is a good replacement for e.g...

But if you confirm you prefer for example I will go for it

Reply:
Yes, that's what I said (but I meant it as a more general comment). I didn't intend to disagree with @pavithraes who suggested "e.g." or "like", and I intended that you should take her suggestion of "e.g." and replace that with "For example"